package.json

```diff
@@ -28,7 +28,7 @@
     "clean": "gts clean",
     "compile": "tsc -p .",
     "prebuild": "npm run clean && npm run check",
-    "build": "npm run clean && npm run check && tsc -p . && npm run test",
+    "build": "npm run prebuild && tsc -p . && npm run test",
```

Comment:
Nice catch. 😄 
Actually, in the `build` it wouldn't require `npm run prebuild` either, as the build system will auto-trigger the `pre` step.